src/server/graphql/models/Team/endMeeting/endMeeting.js

```diff
@@ -48,6 +48,10 @@ export default {
     // RESOLUTION
     const now = new Date();
     const meetingId = meeting.id;
+    const projectsDone = await r.table('Project').getAll(teamId, {index: 'teamId'})
+        .filter({status: 'done'})
+        .filter((project) => project('tags').contains('archived').not())
```

Comment:
this also includes private projects. should those make it into the count? @jordanh